script.js

```diff
@@ -1,52 +1,79 @@
-const container = document.getElementById("container");
-const id = Date. now();
-
-const addToListButton = document.getElementById('add-lists');
-const deleteItemTag = 'span';
-const inputItemTag = 'input';
-
-const saveData = () => {
-  localStorage.setItem('data', container.innerHTML);
+/**  
+ * @type {{ 
+*  items: [
+  *    [
+  *      { 
+  *        text: String, 
+  *        isDone: Boolean 
+  *      }, 
+  *      {
+  *        text: String, 
+ *         isDone: Boolean 
+  *      },
+  *      ...
+  *    ],
+  *    []
+  *  ] 
+  * }}
+  */
+
+const rootState = [];
```

Comment:
Bear in mind the comment on the line 24 and implement state restoring. 
In case if there is no data found use empty array as initial state

Reply:
Fixed)